apps/admin/backend/package.json

```diff
@@ -17,11 +17,11 @@
     "pre-commit": "lint-staged",
     "start": "node ./build/index.js",
     "test": "is-ci test:ci test:watch",
-    "test:ci": "jest --coverage --reporters=default --reporters=jest-junit --maxWorkers=6",
-    "test:coverage": "jest --coverage",
-    "test:debug": "node --inspect-brk $(which jest) --runInBand --no-cache",
+    "test:ci": "TZ=America/Anchorage jest --coverage --reporters=default --reporters=jest-junit --maxWorkers=6",
+    "test:coverage": "TZ=America/Anchorage jest --coverage",
+    "test:debug": "TZ=America/Anchorage jest --inspect-brk $(which jest) --runInBand --no-cache",
```

Comment:
Did we mean to replace `node` with `jest` here?

Reply:
Whoops, that was just a typo